src/arbitrary/constantFrom.ts

```diff
@@ -12,9 +15,13 @@ import { ConstantArbitrary } from './_internals/ConstantArbitrary';
  * @remarks Since 0.0.12
  * @public
  */
-export function constantFrom<TArgs extends any[] | [any]>(...values: TArgs): Arbitrary<TArgs[number]> {
+function constantFrom<T>(...values: T[]): Arbitrary<T>;
+function constantFrom<TArgs extends any[] | [any]>(...values: TArgs): Arbitrary<TArgs[number]>;
```

Comment:
api-extractor complains about the missing JSDoc, please add the same as the one above. I'd probably keep the same `since 0.0.12` as in terms of API it keeps being the same 🤔 (so 100% the same JSDoc should work fine)